outputs.tf

```diff
@@ -7,3 +7,13 @@ output "trail_arn" {
   description = "Cloud trail arn."
   value       = "${element(concat(aws_cloudtrail.cloudtrail.*.arn, list("")), 0)}"
 }
+
```

Comment:
Please add the condition for creating the user as weel. Can you also output the user ARN

Reply:
Added the user ARN, but not sure what you mean regarding the condition. Conditions cannot be added to output variables.